cmd/klocust/apply.go

```diff
@@ -0,0 +1,21 @@
+package klocust
+
+import (
+	"context"
+	"github.com/DevopsArtFactory/klocust/internal/klocust"
+	"github.com/spf13/cobra"
+	"io"
+)
+
+func doApply(_ context.Context, _ io.Writer, _ *cobra.Command, args []string) error {
+	locustName := args[0]
+	return klocust.ApplyLocust(opts.Namespace, locustName)
```

Comment:
`opts.namespace` this value comes from flag but it is accessed directly with variable. I am not sure this is a better approach if there are multiple flags and independent flags for specific commands. 
This is just for brain storming~

Reply:
yeah, I've thought about that. But locustName can be the only one argument without dash(flags).
I think `klocust init hello` is better than `klocust init -name hello` for reading.
But it's not common way for CLI created by golang. It can be changed.

Reply:
Specifying the application name by argument is actually great to use, I think. `--name` is not needed because name itself is required for init command. 

But attributes like namespace, custom variables, etc are optional because we can use default namespace which a user is currently using in the cluster.

So I just think that it could be more flexible to mange these kind of flags in some way (Of course we have to think about it deeply...)  

No need to be fixed in this PR

Reply:
It's good thinking.
Currently, namespace is not required argument and if namespace argument is nil then use user's current namespace.
ref: https://github.com/DevopsArtFactory/klocust/blob/main/internal/kube/namespace.go#L20-L28

But, if there is more arguments in klocust. 
I agree that we need some way to manage flag sets. (And it will be soon with init command for automating initialize.)